File: .claude/skills/database-architecture/examples/javascript/prisma/crud_example.ts
import { PrismaClient } from '@prisma/client'

const prisma = new PrismaClient()

async function main() {
  const user = await prisma.user.create({
    data: { name: 'John', email: 'john@example.com' },
  })

  const users = await prisma.user.findMany()
  console.log(users)

  await prisma.user.delete({ where: { id: user.id } })
}

main()
  .then(async () => await prisma.$disconnect())
  .catch(async (e) => {
    console.error(e)
    await prisma.$disconnect()
    process.exit(1)
  })

File: .claude/skills/database-architecture/examples/python/django/crud_example.py
from django.db import models

class User(models.Model):
    name = models.CharField(max_length=100)
    email = models.EmailField(unique=True)

    class Meta:
        db_table = 'users'

user = User.objects.create(name='John', email='john@example.com')
users = User.objects.all()
User.objects.filter(name='John').delete()